pyproject.toml

```diff
@@ -19,12 +19,9 @@ requires-python = ">=3.10"
 dynamic = ["version"]
 
 dependencies = [
-    #  manages the dependencies for qiskit and tket in a combined fashion
-    "pytket-qiskit>=0.53.0",
-    # lowest version that supports the used pytket AutoRebase pass instead of auto_rebase
-    "pytket>=1.29.0",
     # Qiskit 1.2.1 contains some fixes for importing OpenQASM 3 files
     "qiskit>=1.2.1",
+    "qiskit<=1.4.2",
```

Comment:
You should be able to work around the build failures by replacing these lines with 
```
"qiskit>=1.2.1,<2"
```